tests/kernel/context/testcase.yaml

```diff
@@ -2,3 +2,6 @@ tests:
   kernel.common:
     tags: kernel
     min_ram: 20
+    # FIXME: Remove `qemu_cortex_r5` exclusion once Zephyr SDK 0.11.3 is
+    #        mainlined (see #22904).
+    platform_exclude: qemu_cortex_r5
```

Comment:
Seems cleaner to me to just rush the SDK release than to commit and revert this.

Reply:
That would be nice; though, @galak seems to be busy working on something else, so I am not sure if that is feasible.